oled.js

```diff
@@ -151,7 +151,7 @@ Oled.prototype._waitUntilReady = function(callback) {
     });
   };
 
-  setTimeout(tick(callback), 0);
+  setTimeout(() => { tick(callback) }, 0);
```

Comment:
I propose do not use ES6 in this place to be consistent with other code in this lib. Please use anonymous function.